index.js

```diff
@@ -249,38 +254,40 @@ checkElevated()
             }
         }
 
-        return distro()
-            .catch(() => {});
+        return distro().catch(() => {});
     })
-    .then(dist => {
+    .then((dist) => {
         if (dist && dist.os) {
             config.osdist = dist.os + (dist.release ? ' ' + dist.release : '');
         }
-        return new Promise(resolve => {
-            request('http://169.254.169.254/latest/dynamic/instance-identity/document/', { timeout: 1000, json: true }, function(err, res, body) {
+
+        return new Promise((resolve) => {
+            request('http://169.254.169.254/latest/dynamic/instance-identity/document/', {
```

Comment:
maybe we can put this request url into a const and have a comment about what this is -- this seems like voodoo magic otherwise -- I know, not an actual change of the code, but it would be a nice improvement

Reply:
Yeap - done!